docs/scos/user/intro-to-spryker/whats-new/supported-versions-of-php.md

```diff
@@ -24,15 +24,18 @@ redirect_from:
   - /v6/docs/en/supported-versions-of-php
 ---
 
-**PHP 7.3** is supported until the end of August 2021. Therefore, we strongly recommend switching to **PHP 7.4+** until then.
+# PHP 7
 
-**PHP 8.0** is ready: Since October 2021 we recommend using **PHP 8.0** already, where possible.
+Since April 2022, Spryker requires **PHP 7.4** as the minimum version. We will keep supporting **PHP 7.4** until June 2022.
 
-We plan to start supporting **PHP 8.1** end of 2021 and keep supporting **PHP 7.4** until the end of June 2022.
+# PHP 8
 
-The following graph shows the timelines of support of different PHP versions.
+Since October 2021, Spryker has been compatible with **PHP 8.0**. We recommend using **PHP 8.0** if possible. We plan to support **PHP 8.1** by the end of 2022.
```

Comment:
That must happen sooner, like next month 
But using it remains optional

Reply:
Changed the sentence.